[FIX] auto creation: link existing timesheets

huguesdk opened the pull request:
[task](https://gestion.coopiteasy.be/web#id=7858&view_type=form&model=project.task)

hr_timesheet_auto_creation: link existing timesheets (account.analytic.line) to newly automatically created timesheet sheets.

carmenbianca commented on hr_timesheet_auto_creation/models/hr_timesheet_sheet.py:
```diff
@@ -35,14 +35,20 @@ def create_employee_timesheet(self):
+        sudo_self = self.sudo()
         for employee_id in employee_ids:
-            self.sudo().create(
+            tss = sudo_self.create(
```
I would probably rename this to `sheet` or some such as a more descriptive variable.

huguesdk replied:
indeed! `tss` means “timesheet sheet”, but it’s a little cryptic.

huguesdk committed:
[FIX] auto creation: link existing timesheets

hr_timesheet_auto_creation: link existing timesheets
(account.analytic.line) to newly automatically created timesheet sheets.

robinkeunen reviewed (commented):
Sorry for the late review. 👍 for me, i'm just commenting to warn about asigning a "sudo record" to a variable. When doing so, one must be careful to only use it where sudo is needed since it bypasses all access right. Do not, for example, return the sudo'ed record as it might wander around the code.

Just a warning, it's fine here.

huguesdk commented:
good to know, thanks!
